src/views/ScenarioManager/ScenarioManager.js

```diff
@@ -44,8 +63,12 @@ const ScenarioManager = (props) => {
 
 ScenarioManager.propTypes = {
   datasets: PropTypes.array.isRequired,
+  deleteScenario: PropTypes.func.isRequired,
+  findScenarioById: PropTypes.func.isRequired,
   scenarios: PropTypes.array.isRequired,
-  currentScenario: PropTypes.object.isRequired
+  resetCurrentScenario: PropTypes.func.isRequired,
+  currentScenario: PropTypes.object,
+  userId: PropTypes.string
```

Comment:
UserId can be required?
As scenario deletion is allowed only for scenarios that you have created, the user info is needed.

Reply:
Done :+1: